src/main/java/fi/aalto/cs/apluscourses/ui/ideactivities/ComponentLocator.java

```diff
@@ -21,21 +19,6 @@ public static boolean hasChildOfClass(@NotNull Container c, @NotNull String clas
     return !getComponents(c, comp -> isComponentOfClass(comp, classSubstring)).isEmpty();
   }
 
-  public static boolean hasActionOfClass(@NotNull JButton c, @NotNull String classSubstring) {
-    var action = c.getAction();
-    var actionProperty = c.getClientProperty(CustomComponentAction.ACTION_KEY);
-
-    if (action != null && action.getClass().getName().contains(classSubstring)) {
-      return true;
-    }
-
-    if (actionProperty != null && actionProperty.getClass().getName().contains(classSubstring)) {
-      return true;
-    }
-
-    return false;
-  }
-
```

Comment:
Don't remove it, even if it's unused

Reply:
Can you make a javadoc comment for it please

Reply:
Ok just delete it for now